components/routing/src/Middleware/StartSession.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Torch\Routing\Middleware;
+
+class StartSession
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @param  string|null  $guard
+     * @return mixed
+     */
+    public function handle($request, \Closure $next, $guard = null)
```

Comment:
Please `use` the Closure; and I don't believe `session_start()` needs the preceding slash

Reply:
`session_start()` doesn't _need_ the preceding class, it's just a micro-optimization that I think is a good standard.